template/koa-rest-api/src/listen.ts

```diff
@@ -1,5 +1,3 @@
-import './register';
```

Comment:
It's loaded in app.ts

Reply:
I think this is here to make it obvious (to humans but also to e.g. linters) that the import has side effects and must go first. It's easy to shoot yourself in the foot by importing something else before `./app`, which is part of the reason why I'd like to get rid of this hook soon™️.